core/iwasm/libraries/wasi-nn/test/DOCKERFILE

```diff
@@ -0,0 +1,5 @@
+FROM ubuntu:20.04
+
+RUN apt-get update && apt-get install -y \
+    && build-essential git cmake
+    
```

Comment:
The standard name of the dockerfile is `Dockerfile`.
https://stackoverflow.com/questions/26077543/how-to-name-dockerfiles

Reply:
updated